packages/mobile/src/alert/actions.ts

```diff
@@ -49,6 +57,13 @@ export const showError = (
   )
 }
 
+export const logError = (error: ErrorMessages, i18nOptions?: object): LogAlertAction => ({
```

Comment:
object is actually the right type. its what is used by i18n itself when typing the t function. It cant be any more precise because you can litterally pass any key and value you want and use like `{{keyname}}` in the translation value string

Reply:
You can pass any string value indeed, but there are other options which are different. Using `object` will bypass the real types. See i18next types:
https://github.com/i18next/i18next/blob/f604b34e00d14f74faada327851a47f78603bc5a/index.d.ts#L645